CHANGELOG

```diff
@@ -9,6 +9,7 @@
 * Decidir: Improve handling of error responses from the gateway [naashton] #3594
 * CyberSource: Added support for MerchantInformation CyberSource-specific fields [apfranzen] #3592
 * ePay: Send unique order ids for remote tests [curiousepic] #3593
+* Remove deprecated `rubyforge_project` attribute and tidy up `rake test:local` output [fatcatt316] #3598
```

Comment:
@leila-alderman Let me know your thoughts on this changelog message, por favor 🙇

Reply:
```suggestion
* Remove deprecated `rubyforge_project` and tidy up `rake test:unit` and output [fatcatt316] #3598
```

Although this work was done to clean up `rake test:local`, the changes were technically made to the unit tests.

Reply:
@leila-alderman Updated changelog and rebased off current `master`... this puppy is ready for one final review :salute: